src/forms/TextInput.jsx

```diff
@@ -5,79 +5,62 @@ import cx from 'classnames';
 /**
  * @module TextInput
  */
-class TextInput extends React.Component {
+function TextInput(props) {
```

Comment:
I recommend using an arrow function + static `.displayName` property _or_ a named function, but not a 'bare' arrow function without a name.